default/stringtables/ru.txt

```diff
@@ -281,35 +309,42 @@ FORMAT_LIST_DEFAULT_EMPTY_HEADER
 FORMAT_LIST_DEFAULT_DUAL_HEADER
 Имеются:
 
+FORMAT_LIST_0_ITEMS
+%1%
+
 FORMAT_LIST_1_ITEMS
-%1% %2%
+%1% %2%.
```

Comment:
is adding punctuation here intentional and a good idea?

Reply:
Formatting error will be fixed. Thanks.

Reply:
to be clear, something similar is/was done in various other places as well

Reply:
fixed